jboss-image-streams.json

```diff
@@ -1287,8 +1305,8 @@
                         "annotations": {
                             "description": "JBoss A-MQ 6.2 broker image.",
                             "iconClass": "icon-jboss",
-                            "tags": "messaging,amq,jboss,xpaas",
-                            "supports": "amq:6.2,messaging,xpaas:1.1",
+                            "tags": "messaging,amq,jboss,hidden",
+                            "supports": "amq:6.2,messaging:1.1",
```

Comment:
Looks like the version wasn't nuked when xpaas was deleted.  There should be no version on  messaging.